nmdc_api_utilities/data_staging.py

```diff
@@ -0,0 +1,430 @@
+# -*- coding: utf-8 -*-
+from nmdc_api_utilities.nmdc_search import NMDCSearch
+import logging
+import requests
+from nmdc_api_utilities.auth import NMDCAuth
+from nmdc_api_utilities.decorators import requires_auth
+import json
+
+logger = logging.getLogger(__name__)
+
+
+class JGISequencingProjectAPI(NMDCSearch):
+    """
+    Class to interact with the NMDC API to get JGI samples.
+    """
+
+    def __init__(
+        self,
+        env="prod",
+        auth: NMDCAuth = None,
+        client_id: str = None,
+        client_secret: str = None,
+    ):
+        self.env = env
+        self.auth = auth or NMDCAuth()
+        if client_id and client_secret:
+            self.auth = NMDCAuth(
+                client_id=client_id, client_secret=client_secret, env=self.env
+            )
+        super().__init__(env=env)
+
+    @requires_auth
+    def create_jgi_sequencing_project(
+        self,
+        jgi_sequencing_project: dict,
```

Comment:
non-blocking but it would be good to document the structure of jgi_sequencing_project dict